app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+ belongs_to :role
```

Comment:
indentation issue ?
also please check indentation for this whole file.

Reply:
Updated